packages/common/src/intl/locales/en/app.json

```diff
@@ -56,7 +56,7 @@
   "cmdk.goto-reports": "Go to: Reports",
   "cmdk.goto-master-lists": "Go to: Master Lists",
   "cmdk.placeholder": "Type a command or search",
-  "login.heading": "Simple. Powerful. Pharmaceutical Management.",
+  "login.heading": "Simple.\nPowerful.\nPharmaceutical\nManagement.",
```

Comment:
pleased to find a way around the line wrapping concern 😁

Reply:
This is very strange -- cos regardless of whether you have the line breaks in the string, the rendered HTML is still just:
![Screen Shot 2022-04-06 at 8 37 03 PM](https://user-images.githubusercontent.com/5456533/161932861-d311d69d-8bb9-4068-bfec-fef5adf24386.png)
Not a `<br\>` to be seen! 😲

Anyway -- it works so...

Reply:
super weird.. 🤷